cmd/influx/main.go

```diff
@@ -1,12 +1,12 @@
 package main
 
 import (
-	"flag"
 	"fmt"
 	"os"
 
 	"github.com/influxdata/influxdb/client"
 	"github.com/influxdata/influxdb/cmd/influx/cli"
+	flag "github.com/spf13/pflag"
```

Comment:
nit: I would prefer we don't alias this. It's confusing as when you read the code later you think it's the standard library flag package.